docs/getting-started/os-support.rst

```diff
@@ -27,38 +27,34 @@ ScyllaDB Open Source
-|   5.2                      | |x|  | |x|  | |x|  | |v|  | |v|  | |x|  | |x|  | |v|   | |v|   | |v|     | |v|   |
-+----------------------------+------+------+------+------+------+------+------+-------+-------+---------+-------+
-|   5.1                      | |x|  | |x|  | |v|  | |v|  | |v|  | |x|  | |x|  | |v|   | |v|   | |v|     | |v|   |
-+----------------------------+------+------+------+------+------+------+------+-------+-------+---------+-------+
-|   5.0                      | |x|  | |x|  | |v|  | |v|  | |v|  | |x|  | |x|  | |v|   | |v|   | |v|     | |v|   |
-+----------------------------+------+------+------+------+------+------+------+-------+-------+---------+-------+
-|   4.6                      | |x|  | |v|  | |v|  | |v|  | |x|  | |x|  | |v|  | |v|   | |x|   | |v|     | |v|   |
-+----------------------------+------+------+------+------+------+------+------+-------+-------+---------+-------+
-|   4.5                      | |x|  | |v|  | |v|  | |v|  | |x|  | |x|  | |v|  | |v|   | |x|   | |v|     | |v|   |
-+----------------------------+------+------+------+------+------+------+------+-------+-------+---------+-------+
-|   4.4                      | |x|  | |v|  | |v|  | |v|  | |x|  | |x|  | |v|  | |v|   | |x|   | |v|     | |v|   |
-+----------------------------+------+------+------+------+------+------+------+-------+-------+---------+-------+
-|   4.3                      | |x|  | |v|  | |v|  | |v|  | |x|  | |x|  | |v|  | |v|   | |x|   | |v|     | |v|   |
-+----------------------------+------+------+------+------+------+------+------+-------+-------+---------+-------+
-|   4.2                      | |x|  | |v|  | |v|  | |x|  | |x|  | |x|  | |v|  | |v|   | |x|   | |v|     | |v|   |
-+----------------------------+------+------+------+------+------+------+------+-------+-------+---------+-------+
-|   4.1                      | |x|  | |v|  | |v|  | |x|  | |x|  | |x|  | |v|  | |v|   | |x|   | |v|     | |v|   |
-+----------------------------+------+------+------+------+------+------+------+-------+-------+---------+-------+
-|   4.0                      | |x|  | |v|  | |v|  | |x|  | |x|  | |x|  | |v|  | |x|   | |x|   | |v|     | |x|   |
-+----------------------------+------+------+------+------+------+------+------+-------+-------+---------+-------+
-|   3.x                      | |x|  | |v|  | |v|  | |x|  | |x|  | |x|  | |v|  | |x|   | |x|   | |v|     | |x|   |
-+----------------------------+------+------+------+------+------+------+------+-------+-------+---------+-------+
-|   2.3                      | |v|  | |v|  | |v|  | |x|  | |x|  | |v|  | |v|  | |x|   | |x|   | |v|     | |x|   |
-+----------------------------+------+------+------+------+------+------+------+-------+-------+---------+-------+
-|   2.2                      | |v|  | |v|  | |x|  | |x|  | |x|  | |v|  | |x|  | |x|   | |x|   | |v|     | |x|   |
-+----------------------------+------+------+------+------+------+------+------+-------+-------+---------+-------+
```

Comment:
Why do we list all these old scylla releases, it makes it unclear if you ask me.
Also, why do we list old OS that we don't support anymore.